integration-tests/src/test/java/oracle/kubernetes/operator/utils/PersistentVolume.java

```diff
@@ -19,11 +18,11 @@
   private String dirPath;
 
   /**
-   * Create PV directory and k8s pv and pvc for the domain
+   * Create PV directory and k8s pv and pvc for the domain.
    *
-   * @param dirPath
-   * @param pvMap
-   * @throws Exception
+   * @param dirPath directory path
```

Comment:
this documentation does not tell me anything more that the variable name did

Reply:
Right... again, this was CheckStyle.  It's a CS issue if you have JavaDoc but without text.